gr-blocks/grc/blocks_repeat.block.yml

```diff
@@ -42,7 +42,7 @@ templates:
 cpp_templates:
     includes: ['#include <gnuradio/blocks/repeat.h>']
     declarations: 'blocks::repeat::sptr ${id};'
-    make: 'this->${id} = blocks::repeat::make(${type.size}*${vlen}, ${interp});'
+    make: 'this->${id} = blocks::repeat::make(${type.size})*${vlen}, ${interp});'
```

Comment:
Are you sure about this?

Reply:
I know this looks strange, but if you look below at:
```
translations:
        gr.sizeof_: 'sizeof('
```
The ${type.size} causes a replacement of  **gr.sizeof** with **sizeof(**, which is missing a closing parens.

Reply:
After looking at this more closely, I found that the translation is not needed at all. It is handled implicitly in cpp_top_block.py. So I removed it from this file because the unmodified translation generates non-compilable code (missing a closing parens) i.e. 

`sizeof(gr_complex` 

Note there are at least 40 other YAML files with this broken translation. I'll address those in a separate pull request since this one is already fairly involved.